tests/functional/step_defs/HC-19_test_report_sha.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we need to move this test in the behave path. These are pytest-bdd files, no longer in use.

Reply:
I think these are actually files used by both bdd and behave tests and other behave tests follow this same pattern.

Reply:
No I don't think so, with the behave migration we are not using the `step_defs` any more, so any step definition should be under `steps`

Reply:
Removed it